app/controllers/cloud_network_controller.rb

```diff
@@ -65,6 +65,18 @@ def cloud_network_form_fields
     }
   end
 
+  def cloud_tenants_by_ems
```

Comment:
1. Where should the duplicated code go?  I just want to make sure I'm adhering to code patterns!
2. This seems like a big change for a PR that's intended to fix an Euwe issue, I'm not sure it's safe to do it here.